src/components/board/Stack.vue

```diff
@@ -23,7 +23,7 @@
 
 <template>
 	<div class="stack">
-		<div v-click-outside="stopCardCreation" class="stack--header">
+		<div v-click-outside="stopCardCreation" class="stack--header" :class="{'stack--header--add': showAddCard }">
```

Comment:
That's a weird BEM syntax ^^

Reply:
Indeed :see_no_evil: adjusted with https://github.com/nextcloud/deck/pull/2230/commits/78299b3466dfa84b416d219e4e9e35e64724ff5e